src/app/controllers/VendorController.ts

```diff
@@ -140,9 +121,8 @@ export interface TimeSpanInput {
   to: string;
 }
 
-export interface TimeSpan {
-  from?: string;
-  to?: string;
+export interface TimeSpan extends TimeSpanInput{
```

Comment:
Shouldn't time span input extend from TimeSpan 🤔

Reply:
yes. I agree. And i changed.